cves/CVE-2015-1872.yml

```diff
@@ -64,7 +64,18 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  A lack of validation for the number of components in a Loseless JPEG, that
```

Comment:
Is this meant to be a lossless JPEG? (Loseless may be something different I'm not aware of)